### PR TITLE
Remove Service Workers

### PR DESCRIPTION
**Pull request by Toxiapo**
Ticket: [OE-272](https://jira.nypl.org/browse/OE-272)

This PR removes the service worker and workbox from the repo.

### CODE REVIEW

File: example/axisnow-encrypted.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const AxisNowEncrypted: React.FC<AxisNowEncryptedProps> = (props) => {
 
   return (
     <WebReader
-      injectables={props.injectables}
+      injectablesReflowable={props.injectablesReflowable}
```
Comment (kristojorg) [inline example/axisnow-encrypted.tsx]: This is a breaking change right? The old `injectables` prop no longer works correct? If so, then we will need to bump the version number, provide a little migration guide (just a list of steps) which we can provide to the DRB team

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/web-reader",
-  "version": "3.2.3",
+  "version": "3.3.0",
```
Comment (kristojorg) [inline package.json]: Will likely need to go to v4

File: package.json [hunk 3]
```diff
@@ -84,6 +84,7 @@
     "@babel/core": "^7.12.10",
     "@emotion/jest": "^11.8.0",
     "@parcel/transformer-typescript-tsc": "^2.4.1",
+    "@parcel/transformer-typescript-types": "^2.4.1",
```
Comment (kristojorg) [inline package.json]: Do we need both of these?

**Review by kristojorg — COMMENTED** [review]
Okay this looks good, just the questions about the `injectablesReflowable` change. Did we change that API previously or does both `injectables` and `injectablesReflowable` still work? I don't quite remember

File: example/axisnow-encrypted.tsx (hunk 1 above)
Comment (Toxiapo) [reply example/axisnow-encrypted.tsx]: yup, the `injectables` no longer work, but we already bumped the version to 3 for that upgrade.

Comment (kristojorg) [reply example/axisnow-encrypted.tsx]: Oh I see okay

File: package.json (hunk 3 above)
Comment (Toxiapo) [reply package.json]: I think it was added by mistaken. Deleting now!

**Review by kristojorg — APPROVED** [review]
Looks good to me! When we publish this, let's make sure the change is recorded in the Github Release and ping the DRB team to let them know (though I don't think they used this functionality).

**Review by kristojorg — APPROVED** [review]

**Review by kristojorg — APPROVED** [review]

**Pull request merged**